Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -559,6 +559,51 @@ def get_assets_user_has_access(
             demisto.error(f"Validate API key ReadTimeout error: {str(e)}")
             raise e
 
+    def get_user_context_by_email_address(
+        self, email_address: str
+    ) -> list[dict[str, Any]]:
+        """ Get user context by email address. Returns a dict of user context. """
+        params: dict[str, Any] = {
+            "getTableRequest": {
+                "tableName": "enriched_users_view",
+                "pageSize": 1,
+                "fieldSorts": {
+                    "sorts": []
+                },
+                "fieldFilters": {
+                    "relationship": "FILTER_RELATIONSHIP_OR",
+                    "fieldFilterGroups": {
+                        "fieldFilters": [
+                            {
+                                "relationship": "FILTER_RELATIONSHIP_OR",
+                                "filters": {
+                                    "filters": [
+                                        {
+                                            "field": "email_account",
+                                            "stringContains": {
+                                                "value": f"{email_address}"
+                                            }
+                                        }
+                                    ]
+                                }
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+        try:
+            response = self._http_request(
+                method="POST",
+                url_suffix="/asset-management",
+                timeout=RECO_API_TIMEOUT_IN_SECONDS * 2,
+                data=json.dumps(params),
+            )
+            return extract_response(response)
+        except Exception as e:
+            demisto.error(f"Validate API key ReadTimeout error: {str(e)}")
```

Comment:
If the user does not exist, what happens?
if the API return exception when the user does not exist we are catching the error and raising a timeout error.

Reply:
The best practice here will be to handle it in the main, you have try and except in the main and it will catch the error

Reply:
Great fixed